scripts/ci/run_perftest.sh

```diff
@@ -117,36 +151,73 @@ for scale in 50 100 500; do
     # run ingest tests
     cat ${DATASET_DIR}/$data_fname | $GOPATH/bin/bulk_load_influx $load_opts | jq ". += {branch: \"${INFLUXDB_VERSION}\", commit: \"${TEST_COMMIT}\", time: \"$datestring\", i_type: \"${DATA_I_TYPE}\"}" > $working_dir/test-ingest-$scale_string-batchsize-$batch-workers-$workers.json
 
-    bucket_id=$(curl -H "Authorization: Token $TEST_TOKEN" "http://${NGINX_HOST}:8086/api/v2/buckets?org=$TEST_ORG" | jq -r ".buckets[] | select(.name | contains(\"$db_name\")).id")
-    sleep 1
```

Comment:
In my refactoring, I didn't find this `sleep 1` necessary. Maybe I'm missing something though?

Reply:
Hmm, this could be cruft from prior logic. Either that or I was experiencing problems stopping the daemon for large datasets where the files had not bee completely written to. In either case, it's for the best that this gets removed.